src/main/java/me/ford/iwarp/commands/subcommands/CreateCommand.java

```diff
@@ -77,6 +77,11 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 			return true;
 		}
 
+		if (!warpName.matches(IW.getSettings().getWarpNameFormat())) {
```

Comment:
This is prone to a [PatternSyntaxException](https://docs.oracle.com/javase/8/docs/api/java/util/regex/PatternSyntaxException.html).
Though I'm not really sure that we need to worry about it if we warn users not to touch the option unless they know what they're doing.